charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -34,3 +34,5 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: {{ template "zookeeper-operator.fullname" . }}
+        resources:
+{{ toYaml .Values.resources | nindent 10 }}
```

Comment:
using `nindent` prints an additional line with `helm template`. Could we use `indent` instead?